tests/plugins/test_amp_plugin.py

```diff
@@ -108,7 +108,7 @@ def on_after_backward(self):
         limit_test_batches=2,
         limit_val_batches=2,
         amp_backend='native',
-        distributed_backend='ddp_spawn',
+        distributed_backend='ddp',
```

Comment:
I don't think we can do this, this will make the build hang. It has to remain ddp_spawn

Reply:
spawn has pickle issue but will change back to it

Reply:
I just made a change to the branch, should fix the pickling errors

Reply:
Okay thank you Sean